src/lib/Content/Form/Provider/AbstractGroupedContentFormFieldsProvider.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\ContentForms\Content\Form\Provider;
+
+use Ibexa\Contracts\ContentForms\Content\Form\Provider\GroupedContentFormFieldsProviderInterface;
+use Ibexa\Core\Helper\FieldsGroups\FieldsGroupsList;
+use JMS\TranslationBundle\Model\Message;
+use JMS\TranslationBundle\Translation\TranslationContainerInterface;
+
+abstract class AbstractGroupedContentFormFieldsProvider implements GroupedContentFormFieldsProviderInterface, TranslationContainerInterface
+{
+    protected FieldsGroupsList $fieldsGroupsList;
+
+    protected array $groupContext;
```

Comment:
This is not used in abstract, and can be moved as private property to `GroupedContentFormFieldsProvider`.